App1/App1/App1.csproj

```diff
@@ -6,11 +6,11 @@
 
   <ItemGroup>
     <PackageReference Include="Esri.ArcGISRuntime" Version="100.7.0" />
-    <PackageReference Include="Esri.ArcGISRuntime.Toolkit.Xamarin.Forms" Version="100.7.0" />
+    <PackageReference Include="Esri.ArcGISRuntime.Toolkit.Xamarin.Forms" Version="100.7.1" />
     <PackageReference Include="Esri.ArcGISRuntime.Xamarin.Forms" Version="100.7.0" />
-    <PackageReference Include="Geocortex.Mobile" Version="5.6.0" />
-    <PackageReference Include="Geocortex.Workflow.Forms" Version="5.14.2.184" />
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991221" />
+    <PackageReference Include="Geocortex.Workflow.Forms" Version="5.15.3.342" />
```

Comment:
So I suppose if another Workflow fix gets in we'll need to remember to update the workflow version here again as well.